web-client/src/modules/request/index.ts

```diff
@@ -1,10 +1,92 @@
+import {
+  ContainerOutlined,
+  FileAddOutlined,
+  FileDoneOutlined,
+  FileProtectOutlined,
+  FileSearchOutlined,
+  FileSyncOutlined,
+} from '@ant-design/icons';
+import { uniqueId } from 'lodash';
+import React from 'react';
+import Location from 'react-app-location';
 import { Module } from 'src/types/module';
 
+import { MenuItem } from '../../types/menu-item';
 import ContentPage from './pages/ContentPage';
+import { NewRequestLocation } from './pages/routes/NewRequestRoute/constants';
+
+// TODO (remove link id): ask help to find a better way to keep track of ids
+export const menuItems: Array<MenuItem> = [
+  {
+    id: uniqueId(),
```

Comment:
This is a crucial thing needs to solved by another approach. I'm passing these unique ids to Antd Menu component as a key prop. It uses these keys to find which one is active, clicked, etc. From now, I didn't mark the default selected key on initialization and it's a big problem. I've just an idea to use browser location i.e 'requests/open' as id for menu item whic has a location property which means it's a link. Let's say a user is in '/requests/new' location so we can make it active-selected.

Reply:
I think that is a good idea, however I'm not sure if this would work correctly with translations. for instance if id is /request/new will this work without tricky workarounds when url is /pt/request/new?

Reply:
We don’t have to match by string directly. We could use a regex to check the location vs an exact match.

Reply:
hey, @btuerker , how do you plan to implement the default selected item?
if I remember correctly, we are implementing the` menu item list` and the `menu items` with our own `recursive logic` right?
so I don't see how defining default is going to be a problem here